example-app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        android:extractNativeLibs="true">
```

Comment:
why?

Reply:
This is required by gradle upgrade to version 8.2.0